docs/conf.py

```diff
@@ -286,6 +286,7 @@
     'sphinx': ('https://www.sphinx-doc.org/en/master', None),
     'tox': ('https://tox.readthedocs.io/en/latest', None),
     'twisted': ('https://twistedmatrix.com/documents/current', None),
+    'twistedapi': ('https://twistedmatrix.com/documents/current/api', None),
```

Comment:
I think it'd be better to reiview & merge it separately (https://github.com/scrapy/scrapy/pull/4197)

Reply:
Good point, I added it here because otherwise the `twisted.internet.ssl.Certificate` references do not create links, but there is no harm in that for now.